src/frontend/overlay/src/components/organisms/tickers/Clock.tsx

```diff
@@ -2,9 +2,11 @@ import React from "react";
 import { TextWrap } from "./Text";
 import ContestClock from "../../molecules/Clock";
 
-export const Clock = ({ part }) => {
+export const Clock = ({ tickerSettings, part }) => {
     return <TextWrap part={part}>
-        <ContestClock/>
+        <ContestClock
+            timeZone={tickerSettings.timeZone === "" ? null : tickerSettings.timeZone}
```

Comment:
Why not just make the default null or undefined instead of an empty string?

Reply:
fixed

Reply:
Bug if we have such a situation: we set "timeZone" to utc. Then we set "timeZone" to an empty string. In tickerSettings "timeZone" is now equals to an empty string. So it's not null => we have InvalidDateAndTime.

We can change this part of code:
```js
const dateTime = DateTime.now().setZone(event.target.value);
if (dateTime.isValid || event.target.value === "") {
    setErrorMessage("");
    onChangeFieldEventHandler(setEditData, "timeZone")(event);
} else {
    setErrorMessage(dateTime.invalidReason);
}
```
But I don't think this is a good idea, becase we need to create a new function instead of using onChangeFieldEventHandler. (This necessary for this because if event.target.value is an empty string we can change value to null).

Reply:
Or we can just do this:
```js
if (event.target.value === "") {
    event.target.value = "";
}
```

Reply:
I'm talking mostly about backend behaviour here. As far as i can see this line of code just changes an empty string from the backend to a null

Reply:
Okay I can do something like this:
```kt
override fun toMessage(): ClockTickerMessage {
    if (timeZone != null && timeZone.isEmpty()) {
        return ClockTickerMessage(ClockTickerSettings(part, periodMs, null))
    }
    return ClockTickerMessage(this)
} 
```

Reply:
fixed